### PR TITLE
get all realms

### PR DESCRIPTION
**Pull request by effoeffi**
get all realms added

### CODE REVIEW

File: gocloak.go [hunk 1]
```diff
@@ -1,8 +1,8 @@
 package gocloak
 
 import (
-	"github.com/dgrijalva/jwt-go"
-	"github.com/go-resty/resty/v2"
+	jwt "github.com/dgrijalva/jwt-go"
```
Comment (SVilgelm) [inline gocloak.go]: Unnecessary changes, please remove

File: client.go [hunk 2]
```diff
@@ -987,6 +987,20 @@ func (client *gocloak) GetRealm(token string, realm string) (*RealmRepresentatio
 	return &result, nil
 }
 
+// GetRealms returns top-level representation of all realms
+func (client *gocloak) GetRealms(token string) ([]RealmRepresentation, error) {
```
Comment (SVilgelm) [inline client.go]: Please return the list of pointers, not objects: `[]*RealmRepresentation`

You can take a look at any function what returns a list, for example: https://github.com/Nerzal/gocloak/blob/95a49f04b887503dbfc101150c02e843ee1b6851/client.go#L724

**Comment by SVilgelm** [comment]
Also, update the description of the PR. Thank you for your effort

**Comment by SVilgelm** [comment]
@Nerzal ^ready to merge^

**Comment by Nerzal** [comment]
Thank u very much (:
